app/config/tstvhq/adventures.json

```diff
@@ -81,10 +81,45 @@
                         {
                             "text": "Look for another way out",
                             "nextRoom": ["library"]
-                        }
+                        } 
                     ]
                 }
+
             ]
-        }
+        },
+       {
+            "config": {},
+            "rooms": [ 
+               {
+                    "id": "massAve",
+                    "name": "Mass Ave",
+                    "text": "You are walking along Mass Ave and see the NTB on the corner of Clarendon and Mass Ave. Do you keep walking or turn towards the glowing light away from Dunkin Donuts? Are you man or moth?",
+                    "buttons": [
+                        {
+                            "text": "A mere man",
+                            "nextRoom": ["kitchen"]
+                        },
+                        {
+                            "text": "A mighty moth",
+                            "nextRoom": ["clarendon"]
+                        }
+                    ]
+                },
+        {
+                    "id": "clarendon",
+                    "name": "Mass Ave",
+                    "text": "You mighty moth, you! Are you going to head into Clarendon then? Or do you go ahead for another block or two and see if anyone is around at 24?",
+                    "buttons": [
+                        {
+                            "text": "Clarendon",
+                            "nextRoom": ["clarendon"]
+                        },
+                        {
+                            "text": "24 Please",
+                            "nextRoom": ["24"]
+                        }
+                    ]
+                }
+          }
```

Comment:
still have a mismatched brace. slightly non obvious due to indentation. i can help over chat/zoom